wolfcrypt/src/ecc.c

```diff
@@ -5805,6 +5806,9 @@ static int _ecc_make_key_ex(WC_RNG* rng, int keysize, ecc_key* key,
         if (err == MP_OKAY) {
             err = wc_ecc_curve_load(key->dp, &curve, ECC_CURVE_FIELD_ALL);
         }
+        else {
```

Comment:
Confusing, move up.

Reply:
Agreed. Now that you mention it, that whole section was messy. I believe it is better now.